BambeeHR/pollen
# update/BaseWizard - Show Indicator Label and Left sided Icon

## Issue(s)

- [x] This update is to give the current Base Wizard the ability to look more like the currently Hard Coded in  App, HR Practice Wizard.  This was the original use of the current BaseWizard, but the design was slightly different.  

- [x] This also adds the required set of icons for HR Practices Wizard.

## Description
This PR accomplishes the following:

1. Adding `showIteration` as a prop to the BaseWizard to toggle on the "1 OF 7" or "3 OF 5" label which indicates the current position in the wizard (see screen shot).
2. This PR also includes the ability to add a "left hand side" icon to the NAV Item.

## Screenshot(s)
![Screenshot_20221020_071639](https://user-images.githubusercontent.com/10406785/196973784-86299163-610c-41ab-bbbb-e50e70be642e.png)

## Testing
1. Open Storybook Preview and Navigate to the BaseWizard
2. See the newly added stories: "With Vertical Layout and Slots with Iteration and Icon", "... And Progress"
3. Click through Next to see the Indicator text is shown only for "active" nav item".

## Risk Assessment
LOW - This will not change how the current Wizards function.
